src/components/textbox/textbox.stories.tsx

```diff
@@ -153,6 +153,16 @@ export const WithFieldHelp: ComponentStory<typeof Textbox> = () => {
   return <Textbox label="Textbox" value="Textbox" fieldHelp="Help" />;
 };
 
+export const WithInputHint: ComponentStory<typeof Textbox> = () => {
```

Comment:
**question(non-blocking)**: do we necessarily need this story here given we have demonstrated use of the `inputHint` prop in eariler stories in the docs?